lib/msal-browser/src/interaction_client/SilentCacheClient.ts

```diff
@@ -59,7 +59,7 @@ export class SilentCacheClient extends StandardInteractionClient {
     protected async createSilentFlowClient(serverTelemetryManager: ServerTelemetryManager, authorityUrl?: string, azureCloudOptions?: AzureCloudOptions): Promise<SilentFlowClient> {
         // Create auth module.
         const clientConfig = await this.getClientConfiguration(serverTelemetryManager, authorityUrl, azureCloudOptions);
-        return new SilentFlowClient(clientConfig);
+        return new SilentFlowClient(clientConfig, this.performanceClient);
```

Comment:
Does this need to be a mandatory parameter for SilentFlowClient and RefreshTokenClient ? @jasonnutter @ellymakuba 
This is causing a bunch of tests to fail for #5063